src/platforms/javascript/common/configuration/install-cdn.mdx

```diff
@@ -0,0 +1,91 @@
+
+</markdown>
+</Alert>
+
+### Configure and Verify
+
+`init` Sentry as soon as possible in your app, to make sure it can monitor everything that follows it. Once this is done, all unhandled exceptions are automatically captured by Sentry.
+
+The snippet below includes an intentional error, so you can test that everything is working as soon as you set it up. To view and resolve the recorded error, log into [sentry.io](https://sentry.io) and open your project. Clicking on the error's title will open a page where you can see detailed information and mark it as resolved.
+
+```javascript {tabTitle: CDN}
+Sentry.init({ dsn: "___PUBLIC_DSN___" });
+
+myUndefinedFunction(); // To test your setup
+```
+
+Return now to the [Getting Started](/platforms/javascript/) for our JavaScript SDK to learn about how you can:
+- Capture Errors
+- Automatically Enrich Error Data
+- Set the Release Version
+- Upload Source Maps
+
+## Monitor Performance
+
+Performance Monitoring helps you see everything from macro-level metrics to micro-level spans, and you’ll be able to cross-reference transactions with related issues, customize queries based on your personal needs, and substantially more.
+
+To get started with performance monitoring using Sentry’s JavaScript SDK, install both the `@sentry/browser` and `@sentry/tracing` packages, or link directly to our combined minified bundle via CDN:
+
+<Alert level="note" title="Note"><markdown>
+
+We switched our package name from `@sentry/apm` to `@sentry/tracing`. If you were previously using `@sentry/apm`, please see [Switching to @sentry/tracing](/product/performance/apm-to-tracing/).
+
+</markdown></Alert>
+
+```html {tabTitle: CDN}
+<script
+  src="https://browser.sentry-cdn.com/{{ packages.version('sentry.javascript.browser') }}/bundle.tracing.min.js"
+  integrity="sha384-{{ packages.checksum('sentry.javascript.browser', 'bundle.tracing.min.js', 'sha384-base64') }}"
+  crossorigin="anonymous"
+></script>
+```
+
+Next, initialize the integration in your call to `Sentry.init`:
+
+```javascript {tabTitle: CDN}
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  release: "my-project-name@" + process.env.npm_package_version,
+  integrations: [new Sentry.Integrations.BrowserTracing()],
+  tracesSampleRate: 1.0, // Be sure to lower this in production
+});
+```
+
+Performance data is transmitted using a new event type called “transactions”, which you can learn about in [Distributed Tracing](/product/performance/distributed-tracing). To capture transactions, you must install the performance package and configure your SDK to set the `tracesSampleRate` configuration to a nonzero value. The example configuration above will transmit 100% of captured transactions; lower this value in production to avoid consuming your quota too quickly.
```

Comment:
```suggestion
Performance data is transmitted using a new event type called “transactions,” which you can learn about in [Distributed Tracing](/product/performance/distributed-tracing). To capture transactions, you must install the performance package and configure your SDK to set the `tracesSampleRate` option to a nonzero value. The example configuration above will transmit 100% of captured transactions; lower this value in production to avoid consuming your quota too quickly.
```